examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -185,7 +185,7 @@ def evaluate(model, embed_layer, eval_loader, node_feats):
 
 @thread_wrapped_func
 def run(proc_id, n_gpus, args, devices, dataset, split, queue=None):
-    dev_id = devices[proc_id]
```

Comment:
For cpu runs, the input is already ['cpu']. Why we need to change here?

Reply:
Hi @classicsong thanks for comment.
Regarding you questions please look at line 261 - there is a problem with that condition and str vs int comparison.

Reply:
I c.
Good point.
Can we just change the L514 to send in [-1] ?

Reply:
I don't think so.  For instance `RelGraphEmbedLayer` uses `dev_id` rather than `device`.